source/content.js

```diff
@@ -44,6 +46,10 @@ function onNewTweets(cb) {
 	observeEl('#stream-items-id', cb);
 }
 
+function onSingleTweet(cb) {
+	elementReady('.permalink-tweet-container').then(cb);
```

Comment:
@sindresorhus is there a better/safer way to do this? I don't see anything wrong right now, but not sure if this may bite us in the near future. (I tried to approach this like the `init()` function, but the event wouldn't fire if I used `safeElementReady()` and using `observeEl()` wasn't ideal since the element is dynamically generated)

Reply:
This is very inefficient as `elementReady` just uses a repeated timer to poll for the element. You should instead use `observeEl`. You can observe `body` for attribute changes as `overlay-enabled` is added as a class on `body` when a single tweet is opened.